### PR TITLE
Update utils.go

### LINKED ISSUE #516: Minishift reports 'virtual' as 'virutal'
Just cosmetic - if Minishift fails on startup due to lack of --vm-driver it reports all errors with the word 'virutal' instead of 'virtual'

Example:

E0302 08:53:29.141736   15907 start.go:150] Error starting the VM: Error registering the VM: This virutal machine requires registration. Credentials must either be passed via the environment variables MINISHIFT_USERNAME and MINISHIFT_PASSWORD  or the --username and --password flags

### PR DESCRIPTION
**Pull request by utherp0**
Fix #516  Corrected typo.

### CODE REVIEW

**Comment by budhrg** [comment]
add to whitelist

**Comment by budhrg** [comment]
retest this please

**Comment by budhrg** [comment]
LGTM :+1:

**Comment by praveenkumar** [comment]
Thanks for PR @utherp0 👍 

LGTM

File: pkg/minishift/registration/utils.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 	if registrator != nil {
 		fmt.Println("Registering machine using subscription-manager")
 		if param.Username == "" || param.Password == "" {
-			return errors.New("This virutal machine requires registration. " +
+			return errors.New("This virtual machine requires registration. " +
```
Comment (praveenkumar) [inline pkg/minishift/registration/utils.go]: 👍

**Comment by hferentschik** [comment]
@utherp0, thanks. I merged your pull request. I just adjusted your commit message as well to reference the GitHub issue. Thanks.

**Pull request closed without merging**